JavaScript-foundations-feat-challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_02.js

```diff
@@ -0,0 +1,31 @@
+/* 
+   Spot the bug 🐞
+   Should print: "Most places consider you an adult."... but it doesn't, why?
+
+   Add a comment explaining what is happenning and how to fix it
+*/
+const age = 18;
+
+if (age < 3) {
+  console.log("You're just a baby!");
+} else if (age >= 3) {
+  console.log("You are in elementary school, kid.");
+} else if (age >= 13) {
+  console.log("Nice, you're a teenager!");
+} else if (age >= 18) {
+  console.log("Most places consider you an adult.");
+} else {
+  console.log("What? How did this happen!?");
+}
+
+//The problem is that the age 20, meets with all the adove conditions, thats how you can solve it:
+
```

Comment:
If I execute this solution as follows...
```js
const age = 15;

if (age < 3) {
  console.log("You're just a baby!");
} else if (age < 13) {
  console.log("Nice, you're a teenager!");
} else if (age >= 18) {
  console.log("Most places consider you an adult.");
} else {
  console.log("What? How did this happen!?");
}

```
...output is `What? How did this happen!?`

Have you tested this script before pushing it? Is it properly working or is there any error in it?

⚠️ FIX: spot the 🐛 and fix it